src/request/views.py

```diff
@@ -0,0 +1,44 @@
+import json
+
+from api import settings as api_settings
+from api.utils import success_response
+from rest_framework import generics
+
+from .controllers.create_request_controller import CreateRequestController
+from .models import Request
+from .serializers import RequestSerializer
+
+
+class RequestsView(generics.GenericAPIView):
+    serializer_class = RequestSerializer
+    permission_classes = api_settings.CONSUMER_PERMISSIONS
+
+    def post(self, request):
+        """Create a request."""
+        try:
+            data = json.loads(request.body)
+        except json.JSONDecodeError:
+            data = request.data
```

Comment:
nit: I would suggest we remove this again since our API specifies JSON format or we produce an error if the data is not in JSON format.